integration_tests/test_store_download.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2016 Canonical Ltd
+# Copyright (C) 2016, 2017 Canonical Ltd
```

Comment:
well since I started a review, please make this
`# Copyright (C) 2016-2017 Canonical Ltd`